engine/src/main/java/org/hibernate/validator/internal/engine/ConfigurationImpl.java

```diff
@@ -213,6 +215,25 @@ public final HibernateValidatorConfiguration failFast(boolean failFast) {
 		return this;
 	}
 
+
+	@Override
+	public HibernateValidatorConfiguration setMethodValidationConfiguration(MethodValidationConfiguration mvc) {
+		// the MethodValidationConfiguration cannot be null, it will cause NPE later if it is
```

Comment:
I'd rather fail early with `Contracts#assertNotNull()`, that's what we do in other places, too.